onnxruntime/core/providers/cpu/nn/conv_transpose_attributes.h

```diff
@@ -0,0 +1,242 @@
+/**
+* Copyright (c) 2016-present, Facebook, Inc.
```

Comment:
Is this file renamed from conv_transpose_base.h? If so please do rename and edit to keep the history.